src/lib/requests/baseRequests.js

```diff
@@ -75,6 +75,8 @@ class BaseRequests {
             });
         }
 
+        this.resourceVersions = config.resourceVersions;
```

Comment:
It would be great if we can add this also to the config object of `baseRequests` in `index.d.ts`: https://github.com/mojaloop/sdk-standard-components/blob/master/src/index.d.ts#L105

Even for those not using typescript, the inline definitions are mightly helpful.

Reply:
Oh, sure!